client-js/app/elements/labrad-registry.html

```diff
@@ -25,8 +24,8 @@
       height: 100%;
     }
 
-    .iron-selected {
-      background-color: #ddd;
+    tbody tr:nth-child(1n).iron-selected {
+      background-color: #CCCCCC;
```

Comment:
I prefer to have the newlines to separate things; I find it's easier to read with some whitespace. Of course one can omit whitespace to group closely-related selectors tightly together.

I agree that we're rather inconsistent with formatting here, but I think that's something we can address in subsequent changes.